ci-operator/config/openshift/oadp-operator/openshift-oadp-operator-master.yaml

```diff
@@ -19,7 +19,7 @@ operator:
   bundles:
   - dockerfile_path: build/Dockerfile.bundle
   substitutions:
-  - pullspec: quay.io/konveyor/oadp-operator.v0.2.3:latest
+  - pullspec: quay.io/konveyor/oadp-operator.v0.2.6:latest
```

Comment:
`quay.io/konveyor/oadp-operator:latest`

Reply:
Changed w.r.t this: https://github.com/openshift/oadp-operator/pull/150/files (Pasting for reference)

Reply:
LGTM